neoload/neoload_cli_lib/tools.py

```diff
@@ -90,12 +90,18 @@ def get_named_or_id(name, is_id_, resolver):
     return rest_crud.get(endpoint + "/" + name)
 
 
-def ls(name, is_id_, resolver):
+def ls(name, is_id_, resolver, filter_spec=None, input_params=None):
     endpoint = resolver.get_endpoint()
     if name:
         get_id_and_print_json(get_named_or_id(name, is_id_, resolver))
     else:
-        print_json(rest_crud.get_with_pagination(endpoint))
+        filtering.set_filter(filter_spec, input_params)
+        print_json(
+            filtering.remove_by_last_filter(
```

Comment:
Is it needed to "remove_by_last_filter" here ? "get_with_pagination" already do that !

Reply:
Resolved in 94e75c2

Reply:
This is my approach that provides coverage both from the unit tests and actual usage when running against a real system. If we don't also do this here, coverage over remove_by_last_filter is never produced because we're not actually using the code in get_with_pagination due to monkeypatching. It took me quite a while to find this solution.

Reply:
Ok, so if this line is just to unit test this function, why not do a true unit test of the function "remove_by_last_filter" ? By calling it directly from the unit test ?
If I have the time this morning, I will create the unit test for it